exonum-java-binding-cryptocurrency-demo/frontend/src/plugins/blockchain.js

```diff
@@ -111,13 +110,16 @@ module.exports = {
 
         const body = CreateTransactionProtobuf.encode(data).finish();
 
+        // calculate payload and insert it into buffer
+        Exonum.Uint32.serialize(body.length + SIGNATURE_LENGTH, buffer, 6)
```

Comment:
Don't we include the size of the header in the 'payload length': https://exonum.com/doc/architecture/serialization/#payload-length?